qa-tests-backend/src/main/groovy/objects/Notifiers.groovy

```diff
@@ -75,68 +72,7 @@ class EmailNotifier extends Notifier {
-//            log.debug "checking for messages..."
-//            SearchTerm term = new AndTerm(
-//                    new FromTerm(new InternetAddress(Constants.EMAIL_NOTIFER_SENDER)),
-//                    new SubjectTerm(deployment.name))
-//            notifications = mail.searchMessages(term)
-//            log.debug notifications*.subject.toString()
-//            log.debug "matching messages: ${notifications.size()}"
-//        }
-//        assert notifications.length > 0 // Should be "== 1" - ROX-4542
-//        assert notifications.find {
-//            it.content.toString().toLowerCase().contains("severity: ${policySeverity}") }
-//        assert notifications.find {
-//            containsNoWhitespace(it.content.toString(), "Description:-${policy.description}") }
-//        assert notifications.find {
-//            containsNoWhitespace(it.content.toString(), "Rationale:-${policy.rationale}") }
-//        assert notifications.find {
-//            containsNoWhitespace(it.content.toString(), "Remediation:-${policy.remediation}") }
-//        assert notifications.find { it.content.toString().contains("ID: ${deployment.deploymentUid}") }
-//        assert notifications.find { it.content.toString().contains("Name: ${deployment.name}") }
-//        assert notifications.find { it.content.toString().contains("Namespace: ${deployment.namespace}") }
-//
-//        // Split out so that if recipient email doesn't match, the test will print out all of the emails
-//        // Otherwise it'll print notifications.toString which is unreadable
-//        def recipients = notifications.collect { it.getAllRecipients()*.toString() }
-//        assert recipients.find { it.find { a -> a == this.recipientEmail } }
-//
-//        mail.logout()
-    }
-
-    void validateNetpolNotification(String yaml, boolean strictIntegrationTesting) {
-        // TODO: Replace when https://issues.redhat.com/browse/ROX-12418 is complete
-//        Timer t = new Timer(30, 3)
-//        try {
-//            mail.login()
-//        } catch (Exception e) {
-//            throw new AssumptionViolatedException("Failed to login to GMAIL service... skipping test!: ", e)
-//        }
-//        Message[] notifications = []
-//        while (!notifications && t.IsValid()) {
-//            log.debug "checking for messages..."
-//            SearchTerm term = new AndTerm(
-//                    new FromTerm(new InternetAddress(Constants.EMAIL_NOTIFER_SENDER)),
-//                    new SubjectTerm("New network policy YAML for cluster"))
-//            notifications = mail.searchMessages(term)
-//            log.debug notifications*.subject.toString()
-//            log.debug "matching messages: ${notifications.size()}"
-//        }
-//        assert notifications.length > 0 // Should be "== 1" - ROX-4542
-//        assert notifications.find { containsNoWhitespace(it.content.toString(), yaml) }
-//        mail.logout()
+        log.debug("Not implemented: ROX-12418")
```

Comment:
nit: maybe also leave the comment `TODO(ROX-12418)` since this is somewhat we always do for markers?